README.rst

```diff
@@ -6,9 +6,13 @@
 
 |docs| |devdocs| |usergroup| |developergroup| |anaconda| |mybinder|
 
-MDAnalysis_ is a Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists. 
```

Comment:
We want to keep the sentence as is.

Calling the library "useful" is not helpful – if we didn't think it was useful we wouldn't bother working on it.